PENDING_RELEASE_NOTES.md

```diff
@@ -1,8 +1,14 @@
 
-## Version 15.3.1
+## Version 15.4
 _Date TBD_
 
+### Table Modifications
+* Add table user_certification_body_map to store relationship between users and ACBs
+* Create temporary backup tables for ACL data
+
 ### Data Modifications
 * Change Title & Descriptions for some G1/G2 Macra measures
+* Populate user_certification_body_map based on ACL tables
+* Backup ACL tables and delete that have ACB related information
```

Comment:
The "delete" part isn't happening on this push, right? 

I don't think you need to include data modifications when those tables are being created in the same build. I assume tables that are created get populated during that process, though if you want to make it explicit I'd put that in the "table modification" section. E.g., 
```
* Add table...and ACBs. Populate table at that time
```